keras_app/views/import_json.py

```diff
@@ -107,8 +111,16 @@ def import_json(request):
         name = ''
         class_name = layer.__class__.__name__
         if (class_name in layer_map):
+            # This is to handle wrappers and the wrapped layers.
+            if class_name in ['Bidirectional', 'TimeDistributed']:
+                net[layer.name] = layer_map[class_name](layer)
+                wrapped_layer = layer.get_config()['layer']
+                wrapped_layer['config']['inbound_nodes'] = [[[layer.name]]]                
+                wrapped_layer = jsonLayer(wrapped_layer['class_name'], wrapped_layer['config'], wrapped_layer['config'])
+                net[layer.name]['wrapped_layer'] = wrapped_layer
+                name = layer.name
```

Comment:
You could change this block to something like:
```
net[layer.name+class_name] = layer_map[class_name](layer)
net[layer.name+class_name]['connection']['output'].append(layer.name)
wrapped_layer = layer.get_config()['layer']
wrapped_layer['config']['inbound_nodes'] = [[[layer.name+class_name]]]
wrapped_layer = jsonLayer(wrapped_layer['class_name'], wrapped_layer['config'], wrapped_layer['config'])
net[layer.name] = wrapped_layer
name = layer.name+class_name
```
to have the visualization part start working properly. But, this will not parse the wrapped_layer properly as the parameter names are not corrent in the object you get from `layer.get_config()['layer']`.